i18n/zh/docusaurus-plugin-content-docs/current/reference/sql/show.md

```diff
@@ -381,6 +381,23 @@ SHOW COLUMNS FROM monitor WHERE Field = 'cpu';
 +-------+--------+------+------+---------+-------+---------------+
 ```
 
+列出表中的所有 Region：
+
+```sql
+SHOW REGION FROM monitor;
+```
+
+```sql
++----------------+---------------+------+--------+
+| Table          | Region        | Peer | Leader |
++----------------+---------------+------+--------+
+| monitor        | 4398046511104 |    0 | Yes    |
++----------------+---------------+------+--------+
+```
+
+这是 `INFORMATION_SCHEMA.REGION_PEERS` 的扩展。
```

Comment:
Don't forget to update the chinese doc.

Reply:
resolved in [038669f](https://github.com/GreptimeTeam/docs/pull/1616/commits/038669fe0385cf1be48240068ef7232f976fbe9c)

Reply:
Missing the syntax.

Reply:
Resolved in [de5cb4f](https://github.com/GreptimeTeam/docs/pull/1616/commits/de5cb4fcdb03ff90d7f8cef17cb2cc32ce932cbe)